.travis.yml

```diff
@@ -9,4 +9,4 @@ perl:
     - "5.18"
     - "5.16"
     - "5.14"
-before_install: cpanm --quiet --notest Devel::CheckLib
+before_install: cpanm --quiet --notest Devel::CheckLib Test::More
```

Comment:
Collides with other, specific PR.

Reply:
Yeah I know, this PR includes a drive-by fix. I kind of expect git to be able to resolve the conflict automatically. In any case I don't see any reason to worry about it until git actually complains.